csrc/scheduler/registry_utils.cpp

```diff
@@ -389,12 +389,32 @@ bool requiresForwardViewReplay(Fusion* fusion, ComputeAtMap& ca_map) {
   return false;
 }
 
+namespace {
+bool isSplitOnlyReshapeTransform(Fusion* fusion) {
+  const auto& view_ops = ir_utils::getViewOps(fusion);
+  for (auto view_op : view_ops) {
+    auto tv = view_op->out();
+    for (auto expr : StmtSort::getExprsTo(
+             {tv->getLogicalDomain().begin(), tv->getLogicalDomain().end()})) {
+      if (!expr->isA<Split>()) {
+        return false;
+      }
+    }
+  }
+  return true;
+}
+} // namespace
+
 // Returns if view interferes with how we want to treat the reference, being at
 // least a 2D reduction schedule but maybe a 3D reduction schedule.
 bool reductionInterferingView(
```

Comment:
(Not introduced by this PR)

I wish https://github.com/NVIDIA/Fuser/blob/9c5f006ef99043f7238450c3a96a287687d5144d/csrc/scheduler/reduction.cpp#L926 could be more specific about which View interferes reduction, for debugging. This would require refactoring reductionInterferingView and isSplitOnlyReshapeTransform to generate the error message and propagate it up.

Reply:
will add in a seperate PR.